app/request_lookup.py

```diff
@@ -46,26 +80,68 @@ def get_request_json(data_type, search_value):
         }
     ]
 
-    # Then weight on each individual word
-    if len(search_value.split()) > 1:
-        for word in search_value.split():
-            should_terms.append({
-                "term":
+    if language != 'en':
+        should_terms.append({
+            "term":
                 {
-                    data_type:
+                    language:
                         {
-                            "value": word,
-                            "boost": 2.0
+                            "value": search_value,
+                            "boost": 3.0
                         }
                 }
+
+        })
+
+    # Then weight on each individual word
+    split_search_values = search_value.split()
+    if len(split_search_values) > 1:
+        for word in split_search_values:
+            should_terms.append({
+                "term":
+                    {
+                        "en":
+                            {
+                                "value": word,
+                                "boost": 2.0
+                            }
+                    }
             })
 
+            if language != 'en':
+                should_terms.append({
+                    "term":
+                        {
+                            language:
+                                {
+                                    "value": word,
+                                    "boost": 2.0
+                                }
+                        }
+                })
+
     should_terms.append({
         "match": {
-            data_type + ".trigrams": search_value
+            "en.trigrams": search_value
         }
     })
 
+    if language != 'en':
+        should_terms.append({
```

Comment:
I guess the if the orders of these don't matter, most of the `if` language not `en` could be combined into 1 but I guess how much about readability also.